### ISSUE TITLE
Swapped axes on heatmap in classification notebook

### ISSUE THREAD

**Issue by cgrant42** [MicrosoftDocs/ml-basics#18]
The axes on the heatmap in the classification notebook are incorrect. Predicted species should be on the horizontal axis and actual species should be on the vertical axis. The scikit-learn documentation for the [confusion matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html) states that it returns "Confusion matrix whose i-th row and j-th column entry indicates the number of samples with true label being i-th class and prediced label being j-th class."

**Comment by GraemeMalcolm** [comment]
Nice catch! Fixed throughout.
